app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,31 @@
+class ReviewsController < ApplicationController
+
+  def create
+    @video = Video.find(params[:video_id])
+    @review = @video.reviews.build(params.require(:review).permit(:rating, :review))
+    @review.user = current_user
+    if @review.save
+      flash[:notice] = "Your review has been posted."
+    else
+      flash[:errors] = "You need to enter both a rating and review."
+    end
+    redirect_to video_path(@video)
+  end
+
+  def edit
+    @review = Review.find(params[:id])
```

Comment:
Should the user only be able to edit their own review? If so this should be something like `@review = current_user.reviews.where(id: params[:id]).first`